Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -2695,93 +2692,90 @@ void CClientVehicle::Create()
         if (m_ComponentData.empty())
         {
             // grab our map of components
-            std::map<SString, SVehicleFrame> componentMap = m_pVehicle->GetComponentMap();
-            // get our beginning
-            std::map<SString, SVehicleFrame>::iterator iter = componentMap.begin();
+            auto& componentMap = m_pVehicle->GetComponentMap();
             // loop through all the components.... we don't care about the RwFrame we just want the names.
-            for (; iter != componentMap.end(); iter++)
+            for (const auto& iter : componentMap)
```

Comment:
Please, use structured binding

Reply:
// grab our map of components
std::map<SString, SVehicleFrame> componentMap = m_pVehicle->GetComponentMap();
// loop through all the components.... we don't care about the RwFrame we just want the names.
for (auto iter = componentMap.begin(); iter != componentMap.end(); iter++)
Is that okay?

Reply:
By structured binding I mean `for (const auto& [name, frame] : componentMap)`

Reply:
> for (const auto& [name, frame] :

okay, thanks